src/preset.ts

```diff
@@ -325,6 +328,42 @@ export interface TestPreset extends Preset {
     __generator?: string; // Getting this from the config preset
 }
 
+export interface PackageOutputOptions {
+    debug?: boolean;
+    verbose?: boolean;
+}
+
+export interface PackagePreset extends Preset {
+    configurePreset?: string;
+    inheritConfigureEnvironment?: boolean; // Defaults to true
+    configurations?: string[];
+    generators?: string[];
+    variables?: { [key: string]: string | null | undefined };
+    configFile?: string;
+    output?: PackageOutputOptions;
+    packageName?: string;
+    packageVersion?: string;
+    packageDirectory?: string;
+    vendorName?: string;
+
+    // Private fields
```

Comment:
Didn't notice this earlier when I commented on the same thing in WorkflowPresets. I think, if possible, we should keep these interfaces as exact as possible compared to the Presets docs definition. 

Is there a way to reference these without having "private fields"?

Reply:
If there is a model for this in the build presets or other already existing presets, then I think it's fine. Thanks for pointing that out.